config/zu.py

```diff
@@ -0,0 +1,56 @@
+"Zulu config with language-specific information."
+
+from pynini import *
+
+GRAPHEMES = union("'", "-",
+                  "A", "B", "C", "D", "E", "F", "G", "H", "I",
+                  "J", "K", "L", "M", "N", "O", "P", "Q", "R",
+                  "S", "T", "U", "V", "W", "X", "Y", "Z",
+                  "a", "b", "c", "d", "e", "f", "g", "h", "i",
+                  "j", "k", "l", "m", "n", "o", "p", "q", "r",
+                  "s", "t", "u", "v", "w", "x", "y", "z")
+
+INITIAL_PUNCTUATION = union('"', "'")
+
+FINAL_PUNCTUATION = union("!", '"', ",", ".", ":", ";", "?")
+
+NUMBERS = union("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
+
+HYPHEN = acceptor("-")
+
+SPACE = acceptor(" ")
+
+VOWELS = union("A", "E", "I", "O", "U",
+               "a", "e", "i", "o", "u")
+
+SIGMA_STAR = union(*("[{}]".format(i) for i in range(1, 256))
+                   ).optimize().closure()
+
+NOUN_CLASSIFIERS = union("umu", "um", "u",
+                         "aba", "ab", "abe", "o",
+                         "umu", "um", "u",
+                         "imi", "im",
+                         "ili", "i",
+                         "ama", "ame",
+                         "isi", "is",
+                         "izi", "iz",
+                         "im", "in", "i",
+                         "izim", "izin",
+                         "ulu", "u",
+                         "ubu", "utsh",
+                         "uku", "uk",
+                         "uku", "uk").optimize().closure()
+
+ZU_LOAN_CLASSIFIER = cdrewrite(
```

Comment:
The name of this fix is a little unclear.  I'm not sure if this is acting on loanwords or Zulu words.  Could you add a comment to explain the fix, and also maybe we could rename it to REMOVE_HYPHEN_IN_NOUN_CLASSIFIERS?  Or something of the sort?

Reply:
Fixed!